README.md

```diff
@@ -396,6 +396,7 @@ class ProfileBalloonFactory : Balloon.Factory() {
 .setLayout(@LayoutRes layout: Int)
 .setPreferenceName(value: String)
 .setShowTime(value: Int)
+.setRtlLayout(value: Boolean)
```

Comment:
`setRtlLayout` seems to cause confusion with `setLayout` and even it is a boolean type. 
How about `isRtlSupport`?

Reply:
legit point, fixed